tests/unit/s2n_ecc_evp_test.c

```diff
@@ -345,5 +361,52 @@ int main(int argc, char **argv) {
         }
     }
 
+    /* Test that the client does not negotiate a group that was not
+     * offered in EC preferences */
+    {
+        const struct s2n_security_policy* security_policy = NULL;
+        DEFER_CLEANUP(struct s2n_connection* conn = s2n_connection_new(S2N_CLIENT),
+        s2n_connection_ptr_free);
+        EXPECT_NOT_NULL(conn);
+        EXPECT_SUCCESS(s2n_connection_set_cipher_preferences(conn, "20190802"));
```

Comment:
Probably a good idea to put a comment on here explaining why this security policy was chosen, since their names are so unhelpful.